src/dali/plugin/input/rest/kraken.py

```diff
@@ -254,7 +252,23 @@ def _compute_tx_set(self, trade_history, ledger, processed_transactions) -> List
                         'crypto_received': str(amount) if is_deposit else Keyword.UNKNOWN.value,
                     }
                 )
-                result.append(IntraTransaction(**kwargs))
+                result.append(
+                    IntraTransaction(
+                        plugin=kwargs[Keyword.PLUGIN.value],
```

Comment:
Since kwargs is no longer used for unpacking can you remove it altogether? It just adds one level of indirection that isn't necessary and makes the code harder to read. Use directly the final values instead. E.g.:
```
                        plugin=self.__PLUGIN_NAME,
                        unique_id=Keyword.UNKNOWN.value,
                        ...
```